### PR TITLE
Ports: various dresses/some jean variations and added to loadout. & chaps and corsets added to kinkmate

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
more clothing! also fixed athletic shorts for loadout
also i need clarification on what icons i needed to set, item_color/states. 
idk which ones i needed and which ones i didn't so something's broken

## Why It's Good For The Game
customization options also fixes a bug i made haha

## Changelog
:cl:
add: Ported clothing from RP station, check uniform loadout and kinkmate
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/shorts.dm [hunk 1]
```diff
@@ -35,4 +35,22 @@
...
+	name = "short jean shorts"
+	desc = "Who likes short shorts? we like short shorts!"
+	icon_state = "jean_shorts"
+	item_color = "jean_shorts"
+
+/obj/item/clothing/under/shorts/denimskirt
+	name = "short denim skirt"
+	desc = "Who likes short skirts? we like short skirts!"
+	icon_state = "denim_skirt
+	item_color = "denim_skirt
```
Comment (izzyinbox) [inline code/modules/clothing/under/shorts.dm]: Missing some close quotation marks around here.

File: modular_citadel/code/modules/client/loadout/uniform.dm [hunk 2]
```diff
@@ -78,6 +78,11 @@
 	category = SLOT_W_UNIFORM
 	path = /obj/item/clothing/under/plaid_skirt
 
+/datum/gear/sweptskirt
+	name = "Swept skirt
```
Comment (izzyinbox) [inline modular_citadel/code/modules/client/loadout/uniform.dm]: And here too.
```suggestion
	name = "Swept skirt"
```

**Comment by Ghommie** [comment]
Ok, so apparently icons bot died a day or two ago, and thus I had to download the files to review...
The flamenco, bustle and flower dresses are very nice. 
I'm fine with the most of the rest (the jeans/denim/chaps item sprites are not centered, the sundress item looks like a rag so replace it with the south dir on-mob, yoga pants item sprite too big).
I dislike the green and pink tube dresses, they look like painted tin cans.

Anyhow remember to add digitigrade sprites for all those pants and the crop-top in `uniform_digi.dm` and add a `mutantrace_variation = NO_MUTANTRACE_VARIATION` line for the rest. So we don't end up with missing onmob overlays for damn lizardfolks & co.

**Comment by ghost** [comment]
so should i only make digi variants for the pants/skirts/chaps/croptop?

**Comment by ghost** [comment]
image

**Comment by ghost** [comment]
bruh

**Comment by ghost** [comment]
please don't send it to the shadow realm yet, the conflicts are recent :(

**Comment by ghost** [comment]
AAAAAAAAAAAAAAAAA

**Comment by ghost** [comment]
i think i fixed it

**Comment by ghost** [comment]
I SOILED IT
i'm just gonna remake the PR

**Pull request closed without merging**